src/content/ja/billions/_index.yaml

```diff
@@ -1,150 +1,142 @@
+        label: さまざまなビューポート サイズで良好に動作するコンテンツの設計
+        path: "/web/fundamentals/design-and-ux/responsive/content#viewport"
+      - classname: wf-small-link
+        label: 冗長コンテンツの排除
+        path: "/web/fundamentals/design-and-ux/content/redundant/"
+      - classname: wf-small-link
+        label: モバイル向けの記述
+        path: "/web/fundamentals/design-and-ux/content/write/"
+      - classname: wf-small-link
+        label: AMP HTML の検討
         path: https://www.ampproject.org/
-        classname: wf-small-link
-    - heading: "取引と収益化: グローバルな視点を持つ"
-      description: >
-        ウェブ上の多くのユーザーは支払いシステムを利用できません。全世界の多くのユーザーは銀行口座を持っておらず、クレジットカードを使っていません（8% の米国世帯を含む）。多くの支払いシステムは小さなビューポートや不安定な接続では、うまく機能しません。
+      description: '大型のレイアウト、重いグラフィックスを多用したコンテンツにユーザーはいら立ちを感じ、アプリやサイトを使う気をなくしてしまいます。コンテンツの横幅や縦幅が広すぎると、ユーザーは全体が見えなかったり、そうしたコンテンツを意図的に避けたりします。また、過剰なコンテンツによってページのデータ量が増えるため、データ通信料が高額になります。
+
+'
+      heading: さまざまな端末に対応したコンテンツの構築
+      image_path: images/content.png
+    - buttons:
+      - classname: wf-small-link
+        label: 取引の効率化
+        path: "/web/fundamentals/design-and-ux/principles/#commerce-and-conversion"
+      - classname: wf-small-link
+        label: フォームの効率化
+        path: "/web/fundamentals/design-and-ux/input/forms/"
+      - classname: wf-small-link
+        label: 小さな画面で方向を問わずに使用できるレイアウトの設計
+        path: "/web/fundamentals/design-and-ux/principles/#usability-and-form-factor"
+      description: 'ウェブ上の多くのユーザーは支払いシステムを利用できません。全世界の多くのユーザーは銀行口座を持っておらず、クレジットカードを使っていません（8%
+        の米国世帯を含む）。多くの支払いシステムは小さなビューポートや不安定な接続では、うまく機能しません。
+
+'
+      heading: '取引と収益化: グローバルな視点を持つ'
       image_path: images/monetization.png
-      buttons:
-      - label: 取引の効率化
-        path: /web/fundamentals/getting-started/principles/commerce-and-conversion/
-        classname: wf-small-link
-      - label: フォームの効率化
-        path: /web/fundamentals/design-and-ux/input/forms/
-        classname: wf-small-link
-      - label: 小さな画面で方向を問わずに使用できるレイアウトの設計
-        path: /web/fundamentals/getting-started/principles/#usability-and-form-factor
-        classname: wf-small-link
-    - heading: ローカライズの計画
-      description: >
-        ローカライズにより、新しい市場を開拓できるようになります。多くのウェブサイトは英語のみで公開されていますが、大部分のオンラインユーザーは第一言語が英語ではありません。世界人口の 5.52% のみがネイティブで英語を話し、多くのユーザーは米国やヨーロッパの文化的な習慣に馴染みがなかったり、直感的に理解できなかったりします。英語では簡潔で短い説明やメッセージでも、他の言語では冗長で目障りな場合があります。
-      image_path: images/localization.png
-      buttons:
-      - label: ローカライズに対応したサイトの構築
+    - buttons:
+      - classname: wf-small-link
+        label: ローカライズに対応したサイトの構築
         path: https://support.google.com/webmasters/answer/182192/
-        classname: wf-small-link
+      description: 'ローカライズにより、新しい市場を開拓できるようになります。多くのウェブサイトは英語のみで公開されていますが、大部分のオンラインユーザーは第一言語が英語ではありません。世界人口の
+        5.52% のみがネイティブで英語を話し、多くのユーザーは米国やヨーロッパの文化的な習慣に馴染みがなかったり、直感的に理解できなかったりします。英語では簡潔で短い説明やメッセージでも、他の言語では冗長で目障りな場合があります。
+
+'
+      heading: ローカライズの計画
+      image_path: images/localization.png
+project_path: "/web/_project.yaml"
```

Comment:
Should be at the top

Reply:
@ilyaspiridonov This might be a bug. Can you take a look?